sdc/tests/test_dataframe.py

```diff
@@ -1473,30 +1482,67 @@ def test_impl(df, df2):
         df = pd.DataFrame({'A': np.arange(n), 'B': np.arange(n)**2})
         df2 = pd.DataFrame({'A': np.arange(n), 'C': np.arange(n)**2})
         df2.A[n // 2:] = n
+        hpat_func(df, df2)
         pd.testing.assert_frame_equal(hpat_func(df, df2), test_impl(df, df2))
 
-    @unittest.skip("needs dict typing in Numba")
+    @skip_sdc_jit
+    def test_isin_df2(self):
+        def test_impl(df, df2):
+            return df.isin(df2)
+
+        cfunc = self.jit(test_impl)
+        df = pd.DataFrame({'num_legs': [2, 4], 'num_wings': [2, 0]}, index=['falcon', 'dog'])
+        df2 = pd.DataFrame({'num_legs': [8, 2], 'num_wings': [0, 2]}, index=['spider', 'falcon'])
```

Comment:
Can you extend the data of dfs so that more combinations of results are tested?
That is for index='falcon' you test all values of row in df are in corresponding values in df2 row, but that's only one of the possible results, can you add those that lead to isin False as well? Not sure if that's not already tested with other tests though.